tests/Unit/ItemTest.php

```diff
@@ -4312,6 +4312,93 @@ public static function getTitleDataProvider(): array
                 ,
                 'Item Title',
             ],
+            'Test RSS 0.91 XML and DTD' => [
+<<<XML
+<?xml version="1.0"?>
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>Item Title</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'Item Title',
+            ],
+            'Test RSS 0.91 XML and DTD and external HTML entity' => [
+<<<XML
+<?xml version="1.0"?>
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>caf&eacute;</title>
```

Comment:
Weirdly, the entities from `declare_html_entities` appear to be substituted, even though the `DOCTYPE` within uses `html` as root element. :woman_shrugging:

Reply:
I changed to `rss`